lua/swamp/misc/sh_titles.lua

```diff
@@ -300,30 +302,31 @@ AddTitle({
 }, "Kill %s Kleiners.", "s_kleinerkill")
 
 AddTitle({
-    {100, "Shanker", 0},
-    {1000, "Edge Lord", 0},
-    {5000, "Throat-Neck Slitter", 0},
-    {10000, "American Psycho", 0}
+    {100, "Shanker"},
+    {1000, "Edge Lord"},
+    {5000, "Throat-Neck Slitter"},
+    {10000, "American Psycho", 2000}
```

Comment:
what's with the reward

Reply:
get a free knife after 10k knife kills as a troll